Templates/Per Tenant Extension/.github/workflows/_BuildALGoProject.yaml

```diff
@@ -83,17 +83,17 @@ jobs:
             shell: ${{ inputs.shell }}
             parentTelemetryScopeJson: ${{ inputs.parentTelemetryScopeJson }}
             project: ${{ inputs.project }}
-            get: useCompilerFolder,keyVaultCodesignCertificateName,doNotSignApps
+            get: useCompilerFolder,keyVaultCodesignCertificateName,doNotSignApps,artifact,appFolders,testFolders,bcptTestFolders
```

Comment:
Could we make the name 'artifact' a bit more descriptive? From your PR it looks like it has something to do with the artifact url so maybe "artifacturl"?